app/events/accounts/account_deleted_event.rb

```diff
@@ -5,7 +5,7 @@ def self.create(account)
       account_name: account.name,
       provider:     account.provider_account,
       buyer:        account.buyer,
-      user_id:      account.first_admin.try(:id),
+      user_id:      account.first_admin.try(:id) || account.first_admin_id_on_provider_signup,
```

Comment:
Developer accounts do not have this `first_admin_id` So that might be problematic too no?

Reply:
![image](https://user-images.githubusercontent.com/11318903/49810825-4a4d4380-fd62-11e8-9c03-128f3db29f34.png)

Reply:
So probably it is not `first_admin_id_on_provider_signup` but rather `first_admin_id_on_account_signup`

Reply:
You are right 👍 Pushed 😄